src/etl/affected_genomic_model_etl.py

```diff
@@ -127,170 +134,182 @@ def _process_sub_type(self, sub_type):
+            global_id = agm_record['primaryID']
+            local_id = global_id.split(":")[1]
 
-            if self.testObject.using_test_data() is True:
-                is_it_test_entry = self.testObject.check_for_test_id_entry(globalId)
+            if self.test_object.using_test_data() is True:
+                is_it_test_entry = self.test_object.check_for_test_id_entry(global_id)
                 if is_it_test_entry is False:
                     counter = counter - 1
                     continue
 
-            if agmRecord.get('secondaryIds') is not None:
-                for sid in agmRecord.get('secondaryIds'):
-                    agm_secondaryId_dataset = {
-                        "primaryId": agmRecord.get('primaryID'),
+            if agm_record.get('secondaryIds') is not None:
+                for sid in agm_record.get('secondaryIds'):
+                    agm_secondary_id_dataset = {
+                        "primaryId": agm_record.get('primaryID'),
                         "secondaryId": sid
                     }
-                    agm_secondaryIds.append(agm_secondaryId_dataset)
+                    agm_secondary_ids.append(agm_secondary_id_dataset)
 
-            if agmRecord.get('synonyms') is not None:
-                for syn in agmRecord.get('synonyms'):
+            if agm_record.get('synonyms') is not None:
+                for syn in agm_record.get('synonyms'):
                     syn_dataset = {
-                        "primaryId": agmRecord.get('primaryID'),
+                        "primaryId": agm_record.get('primaryID'),
                         "synonym": syn
                     }
                     agm_synonyms.append(syn_dataset)
 
 
-            if 'crossReference' in agmRecord:
-                crossRef = agmRecord.get('crossReference')
-                crossRefId = crossRef.get('id')
-                local_crossref_id = crossRefId.split(":")[1]
-                prefix = crossRef.get('id').split(":")[0]
-                pages = crossRef.get('pages')
+            if 'crossReference' in agm_record:
+                cross_ref = agm_record.get('crossReference')
+                cross_ref_id = cross_ref.get('id')
+                local_crossref_id = cross_ref_id.split(":")[1]
+                prefix = cross_ref.get('id').split(":")[0]
+                pages = cross_ref.get('pages')
 
                 # some pages collection have 0 elements
                 if pages is not None and len(pages) > 0:
                     for page in pages:
-                        if page == 'Fish' or page == 'genotype' or page == 'strain':
-                            modGlobalCrossRefUrl = ETLHelper.get_page_complete_url(local_crossref_id,
-                                                                                       self.xrefUrlMap, prefix, page)
+                        if page in ['Fish', 'genotype', 'strain']:
+                            mod_global_cross_ref_url = ETLHelper.get_page_complete_url(\
```

Comment:
would it be possible to include the first item in the same line as the function call, instead of the \ -- I find that more readable.  There are several cases of this.